WordPress/Classes/ViewRelated/Gutenberg/Collapsable Header/CollapsableHeaderViewController.swift

```diff
@@ -70,12 +77,21 @@ class CollapsableHeaderViewController: UIViewController, NoResultsViewHost {
             }
         }
     }
+
     private var footerHeight: CGFloat {
         let verticalMargins: CGFloat = 16
         let buttonHeight: CGFloat = 44
         let safeArea = (UIApplication.shared.mainWindow?.safeAreaInsets.bottom ?? 0)
-        return verticalMargins + buttonHeight + verticalMargins + safeArea
+
+        var height = verticalMargins + buttonHeight + verticalMargins + safeArea
+
+        if verticalActionButtons {
+            height += (buttonHeight + selectedStateButtonsContainer.spacing)
```

Comment:
When `verticalActionButtons` is true and only one button is displayed, it seems there is some extra spacing for the footer:

![Simulator Screen Shot - iPhone 8 - 2022-03-31 at 21 34 41](https://user-images.githubusercontent.com/1299411/161081061-7d68090d-b8b2-4e3e-b9f7-2f96bbd376d8.png)

Although I'm guessing `verticalActionButtons` is intended for two buttons, is this something that we need to handle? If it's an unlikely case, then please ignore this. 🙂

Reply:
> Although I'm guessing verticalActionButtons is intended for two buttons, is this something that we need to handle?


Actually, showing the secondary button does need to be optional. I just haven't addressed that scenario yet. So, yea, the spacing will need to be addressed at that time.